addons/block_code/blocks_catalog.gd

```diff
@@ -0,0 +1,38 @@
+extends Object
+
+const BlockDefinition = preload("res://addons/block_code/block_definition.gd")
+const Types = preload("res://addons/block_code/types/types.gd")
+
+static var _created: bool = false
+static var _catalog: Dictionary
```

Comment:
I think since the dictionary defaults to empty, the `_created` boolean is not needed. You can just check `if _catalog: return`.

Reply:
Addressed! Thanks.